src/app/app.routes.js

```diff
@@ -7,13 +7,28 @@ const config = ['$urlRouterProvider','$stateProvider', function(urlRouterProvide
 
     const mainPage = {
         name: 'mainPage',
-        url: '/search?city',
         component: 'mainPage'
     }
 
+    const searchCity = {
+        parent: 'mainPage',
+        name: 'searchCity',
+        url: '/search?city',
+        component: 'containerDays'
+    }
+
+    const errorInfo = {
+        parent: 'mainPage',
+        name: 'errorInfo',
+        url: '/error?status/search?city',
```

Comment:
You should remove the '/search' part because it will be ignored. The url should dbe '/error?status,city'